edsdme/components/PartnerCards.js

```diff
@@ -5,6 +5,16 @@ import './SinglePartnerCard.js';
 const miloLibs = getLibs();
 const { html, LitElement, css, repeat } = await import(`${miloLibs}/deps/lit-all.min.js`);
 
+export function filterRestrictedCardsByCurrentSite(cards) {
+  const currentSite = window.location.pathname.split('/')[1];
+  return cards.filter((card) => {
+    const cardUrl = card?.contentArea?.url;
+    if (!cardUrl) return false;
```

Comment:
is there a reasonable way of validating that cardUrl is a valid URL and new URL(cardUrl) will not fail?